examples/flava/utils.py

```diff
@@ -10,13 +10,15 @@
 
 
 def build_datamodule_kwargs(dm_config: DatasetInfo, training_config: TrainingArguments):
```

Comment:
dm_config type should be TrainingSingleDatasetInfo? Otherwise datamodule_extra_kwargs field won't exist

Reply:
fixed the annotation